presto-manager-controller/src/main/java/com/teradata/prestomanager/controller/RequestDispatcher.java

```diff
@@ -71,7 +75,16 @@ public static Response forwardRequest(String scope, ApiRequester apiRequester, C
         for (URI uri : uriCollection) {
             responseList.add(apiRequester.send(uri));
         }
-        return Response.status(MULTI_STATUS)
-                .entity(new Gson().toJson(responseList)).type(MediaType.APPLICATION_JSON).build();
+
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.enable(SerializationFeature.INDENT_OUTPUT);
+        try {
+            Entity entity = Entity.entity(mapper.writeValueAsString(wrapResponseList(responseList)), MediaType.APPLICATION_JSON_TYPE);
```

Comment:
Split the line 82 into two lines. Its better for reading.
Line 83 can be in a single line.